README.md

```diff
@@ -14,6 +14,26 @@ following links:
 
 [![Build Status](https://travis-ci.org/input-output-hk/cardano-ledger-specs.svg?branch=master)](https://travis-ci.org/input-output-hk/cardano-ledger-specs)
 
+# Repository structure
+
+This repo contains formal (LaTeX) and executable (Haskell model) specs for both
+the Byron and Shelley eras of Cardano. The outline of the specs is as follows:
+
+- [byron](./byron)
+  - [ledger](./byron/ledger)
+    - [formal-spec](./byron/ledger/formal-spec)
+    - [executable-spec](./byron/ledger/executable-spec)
+  - [chain](./byron/chain)
+    - [formal-spec](./byron/chain/formal-spec)
+    - [executable-spec](./byron/chain/executable-spec)
+- [shelley](./shelley)
+  - [design-spec](./shelley/design-spec)
+  - [ledger](./shelley/ledger)
+    - [formal-spec](./shelley/ledger/formal-spec)
+    - [executable-spec](./shelley/ledger/executable-spec)
+    - [dependencies](./shelley/ledger/dependencies)
+  - [chain](./shelley/chain) (for future work on the chain spec)
```

Comment:
The current Shelley ledger spec contains the chain-level validation rules for blocks.  This seemed easier than making a separate document, since otherwise there would have been a lot of duplication, but I realize that this is different than how it was handled for the Byron rules (which has two separate documents).

Reply:
Ah, I see. Should we then drop `chain/` and rename `ledger/` to `specs/`?

Reply:
That works for me! Also, thank you for taking on this reorganization @mhuesch !

Reply:
Another alternative might be to use `chain-ledger` (or `chain-and-ledger`) :man_shrugging:

Reply:
I like `chain-and-ledger` to be extra explicit 🙂